pkg/review/commit_review_status.go

```diff
@@ -143,12 +143,12 @@ type PageInfo struct {
 // A commit is considered properly reviewed as long as there is an associated
 // PR for the commit targeting the repository's main branch with reviewDecision
 // of 'APPROVED'.
-func processCommit(ctx context.Context, commit Commit, gitHubClient *githubv4.Client) *CommitReviewStatus {
+func processCommit(ctx context.Context, gitHubClient *githubv4.Client, commit *Commit) *CommitReviewStatus {
 	logger := logging.FromContext(ctx)
 	logger.InfoContext(ctx, "process commit", "commit", commit)
 
 	commitReviewStatus := CommitReviewStatus{
-		Commit:         commit,
+		Commit:         *commit,
```

Comment:
Can this be a pointer too? Seems like this was probably the reason before also. To avoid the deferencing.

Reply:
So the `Commit` struct is embedded in the `CommitReviewStatus` struct. This is because the columns in the `commit_review_status` BigQuery table are a superset of the columns in the commit view.
https://github.com/abcxyz/github-metrics-aggregator/blob/main/pkg/review/commit_review_status.go#L53-L77

If we change it a pointer I'm not sure how that might afftect how the structs get mapped to BigQuery tables. We can always try it and see what happens.

Reply:
I've updated it to a pointer, if this causes problems then I'll revert it later.